modules/dnn/CMakeLists.txt

```diff
@@ -129,7 +129,12 @@ endif()
 
 set(dnn_runtime_libs "")
 if(INF_ENGINE_TARGET)
-  ocv_option(OPENCV_DNN_IE_NN_BUILDER_2019 "Build with Inference Engine NN Builder API support" ON)  # future: NOT HAVE_NGRAPH
+  set(use_nn_builder OFF)
+  if(TARGET inference_engine_nn_builder OR # custom imported target
+     TARGET IE::inference_engine_nn_builder) # default imported target via InferenceEngineConfig.cmake
```

Comment:
Please add version check for old releases too:
```
  if(TARGET inference_engine_nn_builder  # custom imported target
      OR TARGET IE::inference_engine_nn_builder  # default imported target via InferenceEngineConfig.cmake
      OR INF_ENGINE_RELEASE VERSION_LESS "2020000000"
  )
    set(use_nn_builder ON)
  endif()
```

Reply:
fixed